server/service/endpoint_setup.go

```diff
@@ -7,14 +7,14 @@ import (
 	"github.com/kolide/fleet/server/kolide"
 )
 
-type setupRequest struct {
+type SetupRequest struct {
```

Comment:
I feel like these types should be moved into `package kolide` if they're going to be public.

Reply:
I think I'm going to keep them private and move the client into `package service`

Reply:
But I'm not too sure about this really.